combine.py

```diff
@@ -176,21 +180,31 @@ def main(paths, use_color):
 
     print('Process following {} files: '.format(len(targets)))
     print('\n'.join(['\t{}'.format(f) for f in targets]))
+    print('-'*80)
 
     total = sum(map(FileInfo, targets))
-    with open('result.txt', 'w') as f:
-        total.save_txt(f, use_color)
+    if output is None:
+        for item in total.to_txt(use_color):
+            print(item)
+    else:
+        with open(output, 'w') as f:
+            f.write('\n'.join(total.to_txt(use_color)))
```

Comment:
I feel these changes are ugly. How about implementing as follows?
```python
if output_path is None:
    total.save_txt(stdout, use_color)
else:
    with open(output_path, 'w') as f:
        total.save_txt(f, use_color)
```

Reply:
You can define context manager if you want to unify these `total.save_txt` calls (like as [this site](https://stackoverflow.com/questions/17602878/how-to-handle-both-with-open-and-sys-stdout-nicely)), but I think this solution is too general for this situation.

Reply:
Fixed to output only to stdout and rename function